src/reanimated2/js-reanimated/MapperRegistry.ts

```diff
@@ -1,6 +1,7 @@
 import type { JSReanimated } from './commonTypes';
 import type Mapper from './Mapper';
 
+// ts-prune-ignore-next
```

Comment:
Why isn't it used?

Reply:
Since now, we use mappers from this file: `src/reanimated2/mappers.ts`